lib/actions/index.js

```diff
@@ -0,0 +1,12 @@
+const { UPDATE_TEXT_FIELD } = require('../constants');
```

Comment:
I'd like to build modules in a way such that they can run in nodejs as natively as possible, so that the business logic could be easily ported over to node scripts to help with debugging and reusability